keyboards/hineybush/h65/h65.h

```diff
@@ -26,7 +26,7 @@
  * The second converts the arguments into a two-dimensional array which
  * represents the switch matrix.
  */
-#define LAYOUT_all( \
+#define LAYOUT( \
```

Comment:
```suggestion
#define LAYOUT_all( \
```
Per noroads' suggstions.

Reply:
Is it more conventional to use `LAYOUT_all` even when there's only one layout?

Reply:
I'm assuming there's more than one possible layout, given that there's an `h65` and `h65_hotswap`. `LAYOUT` would probably be preferred for the latter as hotswap boards usually only have a fixed single layout (afaik).

Reply:
That makes sense. I wonder if it'd make sense to combine `h65` and h65_hotswap` given that the only difference is the matrix as mentioned in the original pr 
> Currently using different firmware because the split spacebar matrix assignments are different.

Reply:
If they're different PCBs, they should be different boards.